99-general-concepts/01-debugging/03-breakpoints/README.md

```diff
@@ -1,16 +1,159 @@
-# Breakpoints
+# ¿Cómo debuggear en JS?
 
 * Tipo: `lectura`
 * Formato: `self-paced`
 * Duración: `30min`
 
 ## Objetivos de Aprendizaje
 
-* Representar gráficamente las distintas etapas de un proceso y sus iteraciones,
-  facilitando la comprensión de su funcionamiento.
-* Conocer la utilidad  de los diagramas de flujo para analizar el proceso,
-  proponer ideas, representar los controles, etc.
+* Aprender a depurar a través del Dev Tools
+* Aprender a usar el `debugger statement`
 
 ***
 
-[Contenido de lectura, opcionalmente incluyendo videos, imágenes, ...]
+Para depurar en JavaScript podemos hacer uso de 2 herramientas: DevTools del
+navegador y el debugger statement.
+
+## DevTools
+
+El DevTools (así se llama en Chrome pero existe lo mismo en otros navegadores)
+es una herramienta usada para la inspección y depuración de todo lo que ocurre
+en nuestra aplicación web.
+
+Para acceder a ella existen diversos métodos, uno de los más sencillos es abrir
+la web app que deseas depurar, dar clic derecho en cualquier parte de dicha
+página y seleccionar la opción _Inspect_ o _Inspeccionar_.
+
+Por defecto, abrirá en una pestaña llamada Elements o Elementos; sin embargo,
+para hacer debug de nuestro código JavaScript debemos de cambiar a la pestaña
+_Sources_ o _Fuente_.
+
+Probablemente, al ver el contenido de la pestaña _Sources_, solo verás un
+archivo HTML y ningún archivo JS por más que sepamos que si se está usando. Para
+poder visualizar los archivos JS que usa nuestra web app, solo es necesario
+recargar la página.
+
+¿No te quedó claro lo anterior? No te preocupes, a continuación verás un GIF en
+el cual muestra todo lo mencionado anteriormente.
+
+![Chrome Dev Tools: Sources](https://cdn-images-1.medium.com/max/800/1*JhVFiJNoGZAAmlnk00y6TA.gif)
+
+Para la depuración del código JS, podemos hacer uso de **breakpoints**, que son
+puntos donde queremos que nuestro código se detenga en tiempo de ejecución.
+
+En el siguiente ejemplo, tenemos un código que verifica si el número ingresado
+por un usuario es cero, positivo o negativo.
+
+```javascript
+var number = prompt("Ingrese un número");
```

Comment:
single quotes, according to our new style guide

Reply:
You're right! I'll fix it right now 👍

Reply:
Now it's fixed 😃